.github/doc-label-config.yml

```diff
@@ -0,0 +1,2 @@
+Doc not needed:
+    - '- \[x\]  I have written the necessary rustdoc comments'
```

Comment:
I use this for now, but it's not very accurate. Maybe we can add the third checkbox for this usage.